frontend/src/scenes/data-warehouse/editor/multitabEditorLogic.tsx

```diff
@@ -559,6 +606,23 @@ export const multitabEditorLogic = kea<multitabEditorLogicType>([
         updateDataWarehouseSavedQuerySuccess: () => {
             lemonToast.success('View updated')
         },
+        updateQueryTabState: async (_, breakpoint) => {
+            await breakpoint(100)
```

Comment:
This seems like it's not debouncing relative to my typing speed, and we end up sending to many requests. Slightly worried about out of order state persisting for requests fired in succession to the production backend.